src/commitlint/__version__.py

```diff
@@ -0,0 +1,6 @@
+"""
+This module contains the version information of the current commitlint.
+TODO: automatically bump version through CI.
+"""
+
+__version__ = "0.2.1"
```

Comment:
question: Is `setup.cfg` reusable for this use-case?

Reply:
`setup.cfg` can be used for version information. However, the problem lies in the expense of reading the configuration, making it inefficient. 

Many standard libraries follow similar practices. Here are some references.

mypy: https://github.com/python/mypy/blob/c26f1297d4f19d2d1124a30efc97caebb8c28616/mypy/version.py#L11
pylint: https://github.com/pylint-dev/pylint/blob/2088414ef7353f98b65f61ff211150d9d7816c89/pylint/__pkginfo__.py#L12
celery: https://github.com/celery/celery/blob/7528900d53875bb634f92229ec6ed915c6fc58b6/celery/__init__.py#L20
flake8: https://github.com/PyCQA/flake8/blob/5c52d752e64f210369b37654e7b26ce25e2266d6/src/flake8/__init__.py#L20

Reply:
My point is that if `__version__.py` file is created with version information then, there will be two places where version information exists, `__version__.py` file and `setup.cfg` file. In the case of the above-linked libraries, either they directly put their version in a setup file or reuse the `__version__.py` file in the setup file which we could do as well.

Reply:
I have found the similar problem in the stackoverflow: https://stackoverflow.com/questions/72357031/set-version-of-module-from-a-file-when-configuring-setuptools-using-setup. Let me check it.

Reply:
Reviewed. Thanks @aj3sh . Over to you @subashcs .

Reply:
Looks good